keyboards/bowl/pangea/pngh/keyboard.json

```
@@ -0,0 +1,165 @@
+{
+    "manufacturer": "bowlxgondo",
+    "keyboard_name": "PNG-H Alpha RC1",
+    "url": "",
+    "maintainer": "Gondolindrim",
+    "usb": {
+       "vid": "0x4257",
+       "pid": "0x5054",
+       "device_version": "0.0.1"
+    },
+    "processor": "STM32F401",
+    "bootloader": "stm32-dfu",
+    "matrix_pins": {
+       "cols": ["A8" , "C9" , "C8" , "C7" , "C6" , "B14", "B12", "B10", "B1" , "B0" , "C5" , "C4" , "A7" , "A10", "A6" , "A5" , "A4" , "B7" , "B6" , "C14", "C13"]
+       "rows": ["B5" , "C11", "C12", "D2" , "B3" , "B4"]
+    },
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "build": {
+        "lto": true
+    },
+    "eeprom": {
+        "driver" : "wear_leveling",
+        "wear_leveling": {
+            "driver": "embedded_flash",
+            "backing_size": 4096
+        }
+    },
+    "rgblight":{
+        "led_count": 3,
+        "brightness_steps": 20,
+        "default": {
+            "sat": 0
+        }
```

Sat of 0 isnt going to be significantly different to the default of 255, they are both red.
```suggestion
        "brightness_steps": 20
```